firefox-devtools/original-console-fork
# Batch the message add dispatch for cached messages

Fixes #309 
## Testing instructions
1. Go to Twitter.com
2. Open console

You should still see all the cached messages show

@bgrins the only commit that really is relevant is the last one.
